src/utilities.jl

```diff
@@ -1,6 +1,8 @@
-####
-#### validation
-####
+log(message) = println(now(), " | ", message)
```

Comment:
Why not use the Base logging framework? That's what it's there for.

Reply:
good point, I think my brain just autotyped that out without thinking. ref https://github.com/beacon-biosignals/Onda.jl/pull/59/commits/ca864c4c855ca4e3261032b3bc005082231d3432

Reply:
I meant use `@info` instead of this internal shadowing of `log`

Reply:
Or perhaps even better yet, `@debug`, which can be conditionally enabled.